Frontend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "chase-car-dashboard-frontend",
-  "version": "4.3.0",
+  "version": "5.0.0",
```

Comment:
I don't want to be _too_ nit-picky, but the frontend version bump from this and the data recording changes should be changed so that either:
- We wait to bump the frontend version at all until the python backend refactor is good
- We change the data recording bump to be a major one on both the backend and frontend sides, and this will be a major bump as well. Both changes implement new api calls in the backend and frontend, so it would be a non-backwards compatible change on both ends since they're versioned separately.

I think the first option will be less of a hassle. Just delete the "bump versions" commit that Jack made and remove this change from your "Getting graph to work" commit. Then, hold off on any version updates until `python-backend` is good to merge. Once it is, do a major version bump on all sections of the dashboard.

Reply:
Reverted on new-graph-component. Wait until python-backend is good to rebase onto main before moving version bump to the end of the commit history.